macOSUpgrade.sh

```diff
@@ -315,9 +314,9 @@ loopCount=0
 while [ "$loopCount" -lt 3 ]; do
     if [ -e "$OSInstaller" ]; then
         /bin/echo "$OSInstaller found, checking version."
-        OSVersion=$(/usr/libexec/PlistBuddy -c 'Print :"System Image Info":version' "$OSInstaller/Contents/SharedSupport/InstallInfo.plist")
-        /bin/echo "OSVersion is $OSVersion"
-        if [ "$OSVersion" = "$version" ]; then
+        currentInstallerVersion=$(/usr/libexec/PlistBuddy -c 'Print :"System Image Info":version' "$OSInstaller/Contents/SharedSupport/InstallInfo.plist")
+        /bin/echo "OSVersion is $currentInstallerVersion"
```

Comment:
It would be nice clearer on 'OSVersion'.
How about `/bin/echo "Found installer of macOS ${currentInstallerVersion}."` ?

L.320 should be changed a bit.

Reply:
Thank you for your suggestion.
I fixed in https://github.com/kc9wwh/macOSUpgrade/pull/126/commits/58c3f9a0b8246873bf9f90dd41e8edbed3314e1a.